src/spirv/spirv_module.cpp

```diff
@@ -342,6 +359,44 @@ namespace dxvk {
     
     return this->constComposite(vectorTypeId, args.size(), args.data());
   }
+
+
+  uint32_t SpirvModule::constfReplicant(
+          float                   replicant,
+          uint32_t                count) {
+    std::array<uint32_t, 4> args = {{
+      this->constf32(replicant), this->constf32(replicant),
+      this->constf32(replicant), this->constf32(replicant),
```

Comment:
This is a bit silly; storing the result of `constf32(...)` in a variable and then referencing that would be better.